packages/nimble/R/registration.R

```diff
@@ -17,6 +17,8 @@ if(!exists('.NimbleUseRegistration') || !.NimbleUseRegistration) {
   ## R_setSinglePtrFromSinglePtr = 'setSinglePtrFromSinglePtr'
    C_dwish_chol = 'C_dwish_chol'
    C_rwish_chol = 'C_rwish_chol'
+   C_dinvwish_chol = 'C_dinvwish_chol'
```

Comment:
(Unrelated to this review:) Is this registration needed only so that `dinvwish_chol` can be used in uncompiled nimbleFunctions? I ask because I have not added an entry for `nimOptim` since we use R's builtin `optim`. I just want to make sure we don't need an additional `C_nimOptim`.